src/map/skill.cpp

```diff
@@ -9648,9 +9648,12 @@ int skill_castend_nodamage_id (struct block_list *src, struct block_list *bl, ui
 			if(status_isimmune(bl))
 				break;
 
-			//Remove bonus_script by Dispell
+			// Remove bonus_script by Dispell
```

Comment:
No change

Reply:
Just wanted to bring it into a consistent format since I was working on this part anyway.